modules/users/tests/client/users.client.routes.tests.js

```diff
@@ -257,44 +257,6 @@
         });
       });
 
-      describe('Invalid Route', function () {
```

Comment:
Keep these tests if we end up keeping the client-side routes; based on my comment above.